sdk/extensions/Azure.Extensions.AspNetCore.DataProtection.Keys/api/Azure.Extensions.AspNetCore.DataProtection.Keys.netstandard2.0.cs

```diff
@@ -3,6 +3,6 @@ namespace Microsoft.AspNetCore.DataProtection
     public static partial class AzureDataProtectionKeyVaultKeyBuilderExtensions
     {
         public static Microsoft.AspNetCore.DataProtection.IDataProtectionBuilder ProtectKeysWithAzureKeyVault(this Microsoft.AspNetCore.DataProtection.IDataProtectionBuilder builder, string keyIdentifier, Azure.Core.Cryptography.IKeyEncryptionKeyResolver keyResolver) { throw null; }
```

Comment:
This one is an advanced scenario that requires string because `IKeyEncryptionKeyResolver` is string based.